### ISSUE TITLE
Add option to control hiding

### ISSUE THREAD

**Issue by burn123** [alberto-acevedo/cesium-navigation#33]
Would like to see an option such as `screenSizeHide` that takes a string that is a pixel value, em, rem, etc. This would set the screen size at which the widget will hide. You could also set it to `false` or an empty string and have it always display at any screen size

**Comment by alberto-acevedo** [comment]
Good idea. I would suggest the adding of  properties for pixels screenWidthSizeHide and screenHeightSizeHide.

As an alternative, would it work for you if I add visibility functions to the plugin API to hide/unhide the compass, zoom controls, and distance scale? You could use these functions in your implementation to fully control the visibility.  Thanks.

**Comment by burn123** [comment]
That could also work, but a single option might make it easier for simple uses
